core/modules/views/tests/handlers/views_handler_manytoone.test

```diff
@@ -0,0 +1,616 @@
+   *
+   * @return TaxonomyTrem
+   *   Term with random properties.
+   */
+  protected function createTerm($vocabulary) {
+    $term = entity_create('taxonomy_term', array(
+      'name' => $this->randomName(),
+      'description' => $this->randomName(),
+      'format' => 'plain_text',
+      'vocabulary' => $vocabulary->machine_name,
+    ));
+    taxonomy_term_save($term);
+    return $term;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Create boolean field.
+    $this->fields[0] = array(
+      'field_name' => 'field_bool',
+      'type' => 'list_boolean',
+      'cardinality' => 1,
+      'settings' => array(
+        'allowed_values' => array(
+          0 => '',
+          1 => '',
+        ),
+      ),
+    );
+    $this->fields[0] = field_create_field($this->fields[0]);
+
+    // Create text list field.
+    $this->fields[1] = array(
+      'field_name' => 'field_list',
+      'type' => 'list_text',
+      'cardinality' => FIELD_CARDINALITY_UNLIMITED,
+      'settings' => array(
+        'allowed_values' => array(
+          1 => '1',
+          2 => '2',
+          3 => '3',
+        ),
+      ),
+    );
+    $this->fields[1] = field_create_field($this->fields[1]);
+
+    // Create boolean field instance for post nodes.
+    $instance = array(
+      'field_name' => $this->fields[0]['field_name'],
+      'entity_type' => 'node',
+      'bundle' => 'post',
+      'widget' => array(
+        'type' => 'options_onoff',
+      ),
+    );
+    $this->instances[0][] = field_create_instance($instance);
+
+    // Create text list field instance for post nodes.
+    $instance = array(
+      'field_name' => $this->fields[1]['field_name'],
+      'entity_type' => 'node',
+      'bundle' => 'post',
+      'widget' => array(
+        'type' => 'options_buttons',
+      ),
+    );
+    $this->instances[1][] = field_create_instance($instance);
+
+    // Create boolean field instance for users.
+    $instance = array(
+      'field_name' => $this->fields[0]['field_name'],
+      'entity_type' => 'user',
+      'bundle' => 'user',
+      'widget' => array(
+        'type' => 'options_onoff',
+      ),
+    );
+    $this->instances[0][] = field_create_instance($instance);
+
+    // Create text list field instance for users.
+    $instance = array(
+      'field_name' => $this->fields[1]['field_name'],
+      'entity_type' => 'user',
+      'bundle' => 'user',
+      'widget' => array(
+        'type' => 'options_buttons',
+      ),
+    );
+    $this->instances[1][] = field_create_instance($instance);
+
+    // Create tags field instance for users.
+    $instance = array(
+      'field_name' => 'field_tags',
+      'entity_type' => 'user',
+      'bundle' => 'user',
+    );
+    $this->instances[2][] = field_create_instance($instance);
+
+    // Clear views data cache.
+    $this->clearViewsDataCache();
+
+    // Create two tags.
+    $vocabulary = taxonomy_vocabulary_machine_name_load('tags');
```

Comment:
`taxonomy_vocabulary_machine_name_load()` is [deprecated since 1.0.0](https://api.backdropcms.org/api/backdrop/core%21modules%21taxonomy%21taxonomy.module/function/taxonomy_vocabulary_machine_name_load/1).

```suggestion
    $vocabulary = taxonomy_vocabulary_load('tags');
```